lib/drip/client/shopper_activity.rb

```diff
@@ -75,6 +75,22 @@ def create_product_activity_event(data = {})
         data[:occurred_at] = Time.now.iso8601 unless data.key?(:occurred_at)
         make_json_request :post, "v3/#{account_id}/shopper_activity/product", data
       end
+
+      # Public: Create a checkout activity event.
+      #
+      # options    - Required. A Hash of additional checkout options. Refer to the
+      #                       Drip API docs for the required schema.
+      #
+      # Returns a Drip::Response.
+      # See https://developer.drip.com/#checkout-activity
+      def create_checkout_activity_event(data = {})
+        %i[provider action checkout_id].each do |key|
+          raise ArgumentError, "#{key}: parameter required" if !data.key?(:email) && !data.key?(:person_id)
+        end
```

Comment:
So the goal is to throw 3 exceptions if the `data` hash does not have both an `email` key AND a `person_id` key?

Won't this be confusing if someone calls the method like this?
```ruby
create_checkout_activity_event({provider: "a", action: "b", checkout_id: "c"})
```
It looks like it will raise the following exception:
```ruby
irb(main):007:0> create_checkout_activity_event({provider: "a", action: "b", checkout_id: "c"})
irb(main):008:0*
Traceback (most recent call last):
        7: from /Users/bensandeen/.rbenv/versions/2.7.2/bin/irb:23:in `<main>'
        6: from /Users/bensandeen/.rbenv/versions/2.7.2/bin/irb:23:in `load'
        5: from /Users/bensandeen/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/irb-1.2.6/exe/irb:11:in `<top (required)>'
        4: from (irb):7
        3: from (irb):3:in `create_checkout_activity_event'
        2: from (irb):3:in `each'
        1: from (irb):4:in `block in create_checkout_activity_event'
ArgumentError (provider: parameter required)

Reply:
That was a dumb mistake I didn't see, thanks for pointing it out 😅

Fixed now, tests are passing.